libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -118,7 +130,7 @@ def _parse_chat_history_gemini(
     history: List[BaseMessage],
     project: Optional[str] = None,
     convert_system_message_to_human: Optional[bool] = False,
-) -> List[Content]:
+) -> tuple[Any | None, list[Any]]:
```

Comment:
nits: can it be tuple[Content | Any, list[Content]] maybe?

Reply:
ops, yes it should be

Reply:
should be tuple[Content | None, list[Content]] to be correct